lib/src/components/NxToast/NxToast.scss

```
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  ///nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  //Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  //nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  position: fixed;
+  width: 500px;
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+
+  &.nx-toast--closing {
+    animation: animationOut 0.4s linear;
+  }
+  
+  .nx-alert {
+    margin: var(--nx-spacing-2x) 0 0 0;
+  }
+}
+
+@keyframes animationIn {
+  0% {
+    margin-top: 0;
+    max-height: 0px;
+    right: -500px;
+    opacity: 0;
+  }
+  40% {
+    margin-top: var(--nx-spacing-2x);
+    right: -500px;
+    opacity: 0;
+    max-height:74px;
```

```suggestion
    max-height: 74px;
```
Should also have a comment explaining where this magic number comes from.  It's really unfortunate that we can't make it less magic.

---

> Should also have a comment explaining where this magic number comes from. It's really unfortunate that we can't make it less magic.

Explaining comment added here: [ca49163](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ca49163b4dce7eee38131b2d3ca3b6652eff2c44). I agree that it's not the cleanest either, and unfortunately the only way I could get it to consistently work was to use a rigid pixel count. Perhaps this is something I can look into more after the toast component is done.